daemon/environment.cpp

```diff
@@ -563,7 +566,9 @@ pid_t start_install_environment(const std::string &basename, const std::string &
 
     argv[4] = strdup("-");
     argv[5] = 0;
-    _exit(execv(argv[0], argv));
+    execv(argv[0], argv);
+    log_perror("execv failed");
```

Comment:
That convention (while probably a good idea) is never followed in this entire project.